### ISSUE TITLE
how to upgrade php version

### ISSUE THREAD

**Issue by phoomin2012** [zpanel/installers#44]
How to upgrade php version from 5.3 to 5.4 in zpanel 10.1.1?

**Comment by xtiko14** [comment]
THIS IS HOW TO UPDATE PHP on CENTOS 6.7 oK

1: yum install epel*
2: yum update / yum upgrade
3: 
CentOS and Red Hat Enterprise Linux 6.x
wget http://rpms.famillecollet.com/enterprise/remi-release-6.rpm
sudo rpm -Uvh remi-release-6*.rpm

CentOS and Red Hat Enterprise Linux 7.x
wget http://rpms.famillecollet.com/enterprise/remi-release-7.rpm
sudo rpm -Uvh remi-release-7*.rpm

4: yum update / yum upgrade
5: yum --enablerepo=remi,remi-php56 update
6: yum --enablerepo=remi,remi-php56 upgrade
7: reboot
8: php -v

**Comment by xtiko14** [comment]
It's working very well for me. Enjoy !

**Comment by Basketcasesoftware** [comment]
Ok. Now assume this is not on a Linux system but Windows. How do I upgrade then? Changing OS is NOT an option

**Comment by Basketcasesoftware** [comment]
Seriously? No reply after this long?

**Comment by allebb** [comment]
Probably due to the fact that Zpanel isn't really actively developed anymore; Zpanel has been forked and most people have moved over to http://sentora.org - See you there!! ;)

**Comment by Basketcasesoftware** [comment]
Given that sentora deliberately left off Windows as one its options that is a non-starter. Did you miss that part about "Changing OS is NOT an option". Unless you think shitcanning all my software and doing a few thousand dollars of system changes is a great idea of course.

**Comment by allebb** [comment]
Yes, I did actually miss that part! Nice attitude problem you have there
btw!

**Comment by Basketcasesoftware** [comment]
My request for information was three short sentences that outlined my specifiic issues. Instead of giving a positive answer consistent with my boundary conditions you used it as an advertising plug for a product that was sub-optimum for my needs. Your last response indicates you were more concerned with that advertising than being aware of the specifics of the problem. When you are being that disrespectful and get called on it by the person who you are abusing in that fashion with your negligent reply does not have an "attitude problem". That's a form of victim-blaming.

**Comment by lathif** [comment]
How to Upgrade Php in Zpanel Windows Version...??? help me
